### PR TITLE
Remove performance_schema from parameter group so RDS can manage it.

### PR DESCRIPTION
**Pull request by sureshc**
# Description

We manually reset the `performance_schema` setting in the writer parameter group via the web console and then re-enabled Performance Insights so that RDS would automatically enable performance_schema and related settings on the writer and reader instances on 10/12/2019.

This change removes the explicit configuration of `performance_schema` setting to synchronize our CloudFormation template/stack with the actual configuration of this parameter group.

### Testing
```
$ bundle exec rake stack:data:validate RAILS_ENV=production
Listing changes to existing stack `DATA-production`:
Modify AuroraWriterDBParameters [AWS::RDS::DBParameterGroup] Properties (Parameters)
```

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: aws/cloudformation/db_parameters.yml.erb [hunk 1]
```diff
@@ -183,8 +183,14 @@ AuroraReportingCluster:
...
-  # Enable Performance Schema.
-  performance_schema: 1
+  # Performance Schema and several related Parameter Group settings are enabled automatically on an Aurora Instance
+  # when Performance Insights is enabled on the Instance.  These additional, automatically applied, settings
+  # enable Performance Insights to display detailed Aurora-specific wait events.  Explicitly setting
+  # performance_schema=1 disables the RDS service automatically enabling the related settings that allow for detailed
+  # wait events to be logged in Performance Insights, so we do NOT set it.
+  # https://docs.aws.amazon.com/en_pv/AmazonRDS/latest/AuroraUserGuide/USER_PerfInsights.Enabling.html
+  # Note that our Percona Monitoring server also relies on performance_schema=1 being set automatically by RDS.
+  performance_schema: null
```
Comment (uponthesun) [inline aws/cloudformation/db_parameters.yml.erb]: is there a difference between explicitly setting to null vs. not setting it at all?

Comment (sureshc) [reply aws/cloudformation/db_parameters.yml.erb]: Without explicitly setting it to `null` on the Aurora writer instance, our CloudFormation template rendering logic will [automatically apply](https://github.com/code-dot-org/code-dot-org/blob/bb6588c2b061cb43b97990cc6cd3afef56dcdd80/aws/cloudformation/db_parameters.yml.erb#L12) `performance_schema=1` from the deprecated RDS MySQL production parameter group.

Comment (sureshc) [reply aws/cloudformation/db_parameters.yml.erb]: I'll backlog removal of all RDS MySQL logic (CloudWatch Dashboards, Parameter Groups, etc.).

Comment (wjordan) [reply aws/cloudformation/db_parameters.yml.erb]: > Without explicitly setting it to null on the Aurora writer instance, our CloudFormation template rendering logic will automatically apply performance_schema=1 from the deprecated RDS MySQL production parameter group.

Not sure I understand how a parameter from an unrelated resource would get applied in that way (we don't explicitly reference `Primary` within `AuroraWriter`).

Though I haven't tested it, I would guess that on an existing instance, setting this value to `null` would explicitly change the property to `null`, while removing this key altogether might leave the existing property unchanged.

**Review by wjordan — APPROVED** [review]

**Pull request closed without merging**